localstack/utils/docker_utils.py

```diff
@@ -1508,6 +1569,31 @@ def has_docker(self) -> bool:
         except APIError:
             return False
 
+    def remove_image(self, image: str, force: bool = True):
+        LOG.debug("Removing image %s %s", image, "(forced)" if force else "")
+        try:
+            self.client().images.remove(image=image, force=force)
+        except ImageNotFound:
+            if not force:
+                raise NoSuchImage(image)
+
+    def commit(
+        self,
+        container_name_or_id: str,
+        image_name: str,
+        image_tag: str,
+    ):
+        LOG.debug(
+            "Creating image from container %s as %s:%s", container_name_or_id, image_name, image_tag
+        )
+        try:
+            container = self.client().containers.get(container_name_or_id)
+        except NotFound:
+            raise NoSuchContainer(container_name_or_id)
+        except APIError:
+            raise ContainerException()
+        container.commit(repository=image_name, tag=image_tag)
```

Comment:
Could you move the commit line also into the try block, so APIErrors from this call are mapped to ContainerException as well?

Reply:
Good catch. Fixed in 118e1ca8